packages/apputils-extension/src/shortcuts.tsx

```diff
@@ -62,19 +62,23 @@ export function displayShortcuts(options: IOptions) {
 
   function formatKeys(keys: string[]): JSX.Element {
     const topContainer: JSX.Element[] = [];
-    for (const key of keys) {
+    keys.forEach((key, index) => {
       const container: JSX.Element[] = [];
-      for (const ch of key.split(' ')) {
+      key.split(' ').forEach((ch, chIndex) => {
         container.push(
-          <span className={SHORTCUT_KEY_CLASS}>
+          <span className={SHORTCUT_KEY_CLASS} key={`ch-${chIndex}`}>
             <kbd>{ch}</kbd>
           </span>,
-          <> + </>
+          <React.Fragment key={`fragment-${chIndex}`}> + </React.Fragment>
         );
-      }
-      /*topContainer.push(<span>{container.slice(0, -1)}</span>, <>,</>);*/
-      topContainer.push(<span>{container.slice(0, -1)}</span>, <> + </>);
-    }
+      });
+      topContainer.push(
+        <span key={index} className={`key-${index}`}>
```

Comment:
Did you mean:

```suggestion
        <span key={`key-${index}`}>
```

(may need linting)